sdk/src/main/java/io/matchmore/sdk/monitoring/MatchMonitor.kt

```diff
@@ -0,0 +1,51 @@
+package io.matchmore.sdk.monitoring
+
+import io.matchmore.sdk.AlpsManager
+import io.matchmore.sdk.api.async
+import io.matchmore.sdk.api.models.Device
+import io.matchmore.sdk.api.models.Match
+import java.util.*
+
+interface MatchMonitorDelegate {
+    fun didReceiveMatches(matches: Set<Match>, forDevice: Device)
+}
+
+class MatchMonitor(private val manager: AlpsManager, private val delegate: MatchMonitorDelegate) {
+    var monitoredDevices = mutableSetOf<Device>()
+    var deliveredMatches = mutableSetOf<Match>()
+
+    private var timer: Timer? = null
+
+    fun startPollingMatchesA() {
+        val period: Long = 5000 // repeat every 10 sec.
```

Comment:
5 or 10 ?:) Better to move it to companion object.

Reply:
Good catch :P